translation-definitions/src/main/java/eu/europeana/api/translation/definitions/model/TranslationString.java

```diff
@@ -1,13 +1,13 @@
 package eu.europeana.api.translation.definitions.model;
 
-public class TranslationObj {
+public class TranslationString {
```

Comment:
it is not a String anymore, but a proper object. Probably better to keep Translation or EuropeanaTranslation